packages/guideline-blocks-settings/src/components/Link/LinkSelector/PageLinks.tsx

```diff
@@ -45,8 +45,8 @@ export const PageLinks = ({
 
     if (isLoading) {
         return (
-            <div className="tw-flex tw-justify-center tw-p-4">
-                <LoadingCircle />
+            <div className="tw-flex tw-justify-center tw-h-10 tw-items-center">
+                <LoadingCircle size={LoadingCircleSize.Small} />
             </div>
```

Comment:
This is duplicated with the one here: `packages/guideline-blocks-settings/src/components/Link/LinkSelector/DocumentLinks.tsx` can we have it as an own component and just reuse it?